pipelines/_ingredients/DECam/DRP.yaml

```diff
@@ -199,7 +202,8 @@ subsets:
     subset:
       - selectGoodSeeingVisits
       - selectDeepCoaddVisits
-      - makeWarp
+      - makeDirectWarp
+      - makePsfMatchedWarp
```

Comment:
@leeskelvin I haven't tested that these changes that I am making to DRP pipeline are okay on DECam beyond CI passing. I've done more detailed tests in DM-45426 on HSC and DC2 datasets. Is there a coadd dataId I should run this on on DECam data to make sure that the coadds are largely the same even after I swap out the warping tasks?